man/udevadm.xml

```diff
@@ -727,6 +727,48 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><option>-L</option></term>
```

Comment:
I don't thin this needs a short option. Short options are for stuff frequently needed on the cmdline I'd say. But this one doesn#t qualify afaics.